libs/external-router/driver/magento/src/model/url-resolution-response.ts

```diff
@@ -0,0 +1,5 @@
+import { MagentoUrlResolver } from './url-resolver';
+
+export interface MagentoUrlResolutionResponse {
```

Comment:
The name of this interface should now be `MagentoUrlResolverResponse`. The directory path should be `src/model/url-resolver-response.ts`

Reply:
7cc9522a